tekton/publish.yaml

```diff
@@ -198,6 +198,7 @@ spec:
             do
               crane cp ${IMAGE_WITH_SHA} ${REGION}.${IMAGE_WITHOUT_SHA_AND_TAG}:$TAG
             done
+            echo ${REGION}.$IMAGE_WITH_SHA, >> $(results.IMAGES.path)
```

Comment:
thanks @priyawadhwa, we can refactor and move `echo` out of the `if..else` block? 🤔 https://github.com/tektoncd/chains/pull/242#discussion_r717138844

Reply:
👍🏽 done! :)